src/mjolnir/graphenhancer.cc

```diff
@@ -1620,7 +1620,7 @@ void enhance(const boost::property_tree::ptree& pt,
         }
 
         // Update the named flag
-        auto names = tilebuilder->edgeinfo(&directededge).GetNamesAndTypes(true);
+        auto names = tilebuilder->edgeinfo(&directededge).GetNamesAndTypes();
```

Comment:
this `true` seems to be the reason of the issue with isochrones
I don't think we should consider edges with z-level to be "named", but I'm not quite confident if we should do it for edges which have tunnel or bridge names. Probably yes. So I think we can have 2 options here:
1. Simply use `false`(i.e. edge is named only if there is street name)
2. Change API of `EdgeInfo` a little bit to have convenient method to know if we have anything there which allows to call this node "named"(probably simply have `bool named() const` method there too).
WDYT? @kevinkreiser

Reply:
> if we have anything there which allows to call this node "named"

i.e. if we have tunnel name or street name there inside

but most likely if we have tunnel name we would have a street name too 🤔

Reply:
good find! ok so i agree, that no, zlevel has nothing to do with a name and that yes, bridge and tunnel are named. 

i'm not sure i like any of the suggestions though unless im not understanding them correctly. the point of that function call is to get all names for an edge. previously we only had bridge and tunnel as special tag values and they are indeed names. now we are finally realizing the point of the tag value feature and adding a non name. what i would suggest is to modify that function to return only names. we can make a `const unordered_set<TaggedValue> name_tags{ kBridge, kTunnel};` which we can use in that function to look up which tagged values should be included in this return.

Reply:
https://github.com/valhalla/valhalla/pull/3261/files#diff-9be7fe3b13039175d64773b5e79235508b5875285151c60f376bef02f8efbfe1R106

we could add to this `if` a check if the tag enum is present in the `name_tags` set and only if it is do we push it back

Reply:
@gknisely does this make sense with the phoneme work?